reference/concepts/actor_model.md

```diff
@@ -1,7 +1,7 @@
 # Actor Model
 
-> The _actor model_ is a computational model of concurrency where an _actor_ is the computational primitive.  In response to messages sent by other _actors_, the _actor_ can: make local decisions, create more actors, send more messages, determine how to respond to the next message.  They can manage their own private state, and can affect other _actors_ only by sending a message.
+> The _actor model_ is a computational model of concurrency where an _actor_ is the computational primitive. In response to messages sent by other _actors_, the _actor_ can: make local decisions, create more actors, send more messages, determine how to respond to the next message. They can manage their own private state, and can affect other _actors_ only by sending a message.<sup>1</sup>
 
-## Sources
+---
```

Comment:
Could we add this to the FAQ as well, to standardise on a format?

Reply:
Done!